(8.0) PXC-4394 Misleading SST version comparison message

aybek opened the pull request:
https://perconadev.atlassian.net/browse/PXC-4394

Added a validation to ensure the pxb version could be retrieved before performing the version comparison.

kamil-holubicki commented on scripts/wsrep_sst_xtrabackup-v2.sh:
```diff
@@ -1753,6 +1753,16 @@ if [[ ! -x $XTRABACKUP_24_PATH/bin/$XTRABACKUP_BIN ]]; then
+
+if [[ -z "$XB_2x_VERSION" ]]; then
+    wsrep_log_error "******************* FATAL ERROR ********************** "
+    wsrep_log_error "* Cannot determine the xtrabackup 2.x version. "
```
I see that this file is not consistent in error log messages.
Please remove the leading asterisk from this line, or add the leading asterisk to all lines.
I suggest the 1st solution (remove).

kamil-holubicki commented on scripts/wsrep_sst_xtrabackup-v2.sh:
```diff
@@ -1779,6 +1789,16 @@ if [[ ! -x $XTRABACKUP_80_PATH/bin/$XTRABACKUP_BIN ]]; then
+
+if [[ -z "$XB_8x_VERSION" ]]; then
+    wsrep_log_error "******************* FATAL ERROR ********************** "
+    wsrep_log_error "* Cannot determine the xtrabackup 8.x version. "
```
likewise

aybek committed:
PXC-4394 Misleading SST version comparison message
https://perconadev.atlassian.net/browse/PXC-4394

Added a validation to ensure the pxb version could be retrieved before performing the version comparison.

aybek commented:
https://pxc.cd.percona.com/view/8.0%20parallel%20MTR/job/pxc-8.0-pipeline-parallel-mtr/3142/testReport/
